include/otf2xx/definition/detail/base.hpp

```diff
@@ -65,7 +65,6 @@ namespace definition
         class base
         {
         public:
-            using reference_type = otf2::reference<Def>;
```

Comment:
The Def template param can now be removed.

Reply:
It can't. Still needed for the weak_ref template parameter